src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -358,6 +359,26 @@ private static void DisplayFullInformation(TestResult result)
 
         #region Event Handlers
 
+        /// <summary>
```

Comment:
Make sure this part of the vstest.console help

Reply:
Done.